app/views/layouts/_navigation_links.html.erb

```diff
@@ -5,6 +5,9 @@
     <% unless current_page?(authenticated_root_url) %>
       <li><%= link_to 'Dashboard', authenticated_root_path %></li>
     <% end %>
+    <% if current_user.admin? %>
+      <li> <%= link_to 'Create User', new_user_url %></li>
```

Comment:
I ask because this is something that has bitten me in the behind a lot recently; can we use `new_user_path` over `new_user_url` here?